### ISSUE TITLE
Scrubbers and vents off on instanced ships loaded midround

### ISSUE THREAD

**Issue by someone543** [BeeStation/NSV13#2395]
## Round ID: N/A
Tested on master

## Testmerges: N/A
Tested on master

## Reproduction:
Loading a ship midround (Loading before the round starts has no issues) using instance_overmap causes all scrubbers and vents on the ship to be off. Other issues could have been caused, but this is the most noticeable.
1. Wait for a round to start
2. Use instance_overmap to load any ship in
3. See all scrubbers and vents start off on the ship

![MidroundInstance](https://user-images.githubusercontent.com/95106800/232182598-3d781c1d-8292-471a-9601-a17455284cf7.PNG)

**Comment by Bokkiewokkie** [comment]
Yeah we've seen this a couple of times over the years, the only constant seems to be that it just works:tm: on some people's machines and not on others making it terrible to debug
